src/MatrixClientPeg.js

```diff
@@ -94,6 +95,8 @@ class MatrixClientPeg {
     }
 
     async start() {
+        StorageManager.trackStores(this.matrixClient);
```

Comment:
This should happen after the loop below because `store` gets reassigned

Reply:
Ah right, good catch! 😁